src/util/colorScale.ts

```diff
@@ -1,24 +1,42 @@
-import chroma from 'chroma-js';
-import IRoute from '~/models/IRoute';
 
 class ColorScale {
-    public static getColorMap(routes: IRoute[]) {
-        const colorMap = new Map();
+    public static allColors = [
+        '#e6194B',
+        '#3cb44b',
+        '#ffe119',
+        '#4363d8',
+        '#f58231',
+        '#42d4f4',
+        '#f032e6',
+        '#fabebe',
+        '#469990',
+        '#e6beff',
+        '#9A6324',
+        '#fffac8',
```

Comment:
#fffac8 quite hard to see on the map. Maybe darken it a bit?

Reply:
Good point! I think we just drop it from the list..